projects/packages/sync/src/modules/class-woocommerce.php

```diff
@@ -61,16 +61,52 @@ class WooCommerce extends Module {
 	private $order_item_table_name;
 
 	/**
-	 * The table in the database.
+	 * The table name.
 	 *
 	 * @access public
 	 *
 	 * @return string
+	 * @deprecated since $$next-version$$ Use table() instead.
 	 */
 	public function table_name() {
+		_deprecated_function( __METHOD__, '$$next-version$$', 'Automattic\\Jetpack\\Sync\\WooCommerce->table' );
 		return $this->order_item_table_name;
 	}
 
+	/**
+	 * The table in the database with the prefix.
+	 *
+	 * @access public
+	 *
+	 * @return string|bool
+	 */
+	public function table() {
+		global $wpdb;
+		return $wpdb->prefix . 'woocommerce_order_items';
```

Comment:
I was wondering if we still need to set `order_item_table_name` in the constructor? Maybe we could refactor the methods that use ii, eg `expand_order_item_ids` to use our new `table` method instead?

Reply:
You are right, but I think we can do that once we remove the deprecated function.